content/02.main-text.md

```diff
@@ -20,8 +20,54 @@ Although we requested that some authors participate for their specific expertise
 `TODO: confirm "most"`
 To coordinate this effort, we developed a manuscript writing process using the Markdown language, the GitHub software development platform [@url:https://github.com/greenelab/deep-review/], and our new Manubot tool [@url:https://github.com/greenelab/manubot-rootstock; @url:https://github.com/greenelab/manubot] for automating manuscript generation.
 
-
-### Collaborative writing and reviewing
+### Manubot
+
+We developed Manubot, a system for writing scholarly manuscripts via GitHub.
+With Manubot, manuscripts are written as plain-text markdown files, which is well suited for version control using git.
+The markdown standard itself provides limited yet crucial formatting syntax, including the ability to embed images and format text via bold, italics, hyperlinks, headers, inline code, codeblocks, blockquotes, and numbered or bulleted lists.
+In addition, Manubot relies on Pandoc's extended markdown to enable citations, tables, captions, and equations specified using the popular TeX math syntax.
+
+Manubot includes an additional layer of citation processing, currently unique to the system.
+All citations point to a standard identifier, for which Manubot automatically retrieves bibliographic metadata.
+Currently, citations to DOIs (Digital Object Identifiers), PubMed identifiers, arXiv identifiers, and URLs (web addresses) are supported.
+Metadata is retrieved using DOI [Content Negotiation](https://citation.crosscite.org/docs.html), NCBI's [Citation Exporter](https://www.ncbi.nlm.nih.gov/pmc/tools/ctxp/), the [arXiv API](https://arxiv.org/help/api/index), or [Greycite](http://greycite.knowledgeblog.org/) [@arxiv:1304.7151v1].
+Metadata is exported to [CSL JSON Items](http://citeproc-js.readthedocs.io/en/latest/csl-json/markup.html#items), an open standard that's widely supported by reference managers [@doi:10.1007/978-3-319-00026-8_8; @doi:10.1080/02763869.2012.641841].
+In cases where automatic retrieval of metadata fails or produces incorrect references -- which is most common for URL citations -- users can manually provide the correct CSL JSON.
+
+The Manubot formats bibliographies according to a [Citation Style Language](http://citationstyles.org/) (CSL) specification.
+As a result, users can choose from thousands of existing CSL styles or use Manubot's default style.
+Styles define how references are constructed from bibliographic metadata, controlling layout details such as the max number of authors to list per reference.
+Thousands of journals have [predefined styles](http://editor.citationstyles.org/searchByName/).
+As a result, adopting the specific bibliographic format required by a journal usually just requires specifying the style's source URL in the Manubot's configuration.
+
+Manubot uses [Pandoc](https://pandoc.org/) to convert manuscripts from markdown source to HTML (and optionally DOCX) output.
```

Comment:
Move link to the first mention of Pandoc.

Can this be simplified to `to HTML, PDF, and optionally DOCX output`?

Reply:
Will consider. Technically PDF is converted from the HTML using WeasyPrint. But that may be an irrelevant detail.